recaptcha_enterprise/snippets/src/main/java/recaptcha/mfa/CreateMfaAssessment.java

```diff
@@ -0,0 +1,158 @@
+import com.google.recaptchaenterprise.v1.AccountVerificationInfo.Result;
+import com.google.recaptchaenterprise.v1.Assessment;
+import com.google.recaptchaenterprise.v1.CreateAssessmentRequest;
+import com.google.recaptchaenterprise.v1.EndpointVerificationInfo;
+import com.google.recaptchaenterprise.v1.Event;
+import com.google.recaptchaenterprise.v1.ProjectName;
+import com.google.recaptchaenterprise.v1.TokenProperties;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+public class CreateMfaAssessment {
+
+  public static void main(String[] args)
+      throws IOException, NoSuchAlgorithmException, InvalidKeyException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Google Cloud Project ID.
+    String projectId = "PROJECT_ID";
+    // Site key obtained by registering a domain/app to use recaptcha services.
+    // See, https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages
+    String recaptchaSiteKey = "SITE_KEY";
+    // The token obtained from the client on passing the recaptchaSiteKey.
+    // To get the token, integrate the recaptchaSiteKey with frontend. See,
+    // https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
+    String token = "TOKEN";
+    // The action name corresponding to the token.
+    String recaptchaAction = "ACTION";
+    // Email id of the user to trigger the MFA challenge.
+    String emailId = "foo@bar.com";
+    // Phone number of the user to trigger the MFA challenge.
+    String phoneNumber = "+11111111111";
+
+    // Create hashedAccountId from user identifier.
+    // It's a one-way hash of the user identifier: HMAC SHA256 + salt.
+    String userIdentifier = "Alice Bob";
+    // Change this to a secret not shared with Google.
+    final String hmacKey = "SOME_INTERNAL_UNSHARED_KEY";
+    // Get instance of Mac object implementing HmacSHA256, and initialize it with the above
+    // secret key.
+    Mac mac = Mac.getInstance("HmacSHA256");
+    mac.init(new SecretKeySpec(hmacKey.getBytes(StandardCharsets.UTF_8),
+        "HmacSHA256"));
+    byte[] hashBytes = mac.doFinal(userIdentifier.getBytes(StandardCharsets.UTF_8));
+    ByteString hashedAccountId = ByteString.copyFrom(hashBytes);
+
+    createMfaAssessment(projectId, recaptchaSiteKey, token, recaptchaAction, hashedAccountId,
+        emailId, phoneNumber);
+  }
+
+  // Creates an assessment to obtain Multi-Factor Authentication result.
+  // If the result is unspecified, sends the request token to the caller to initiate MFA challenge.
+  // See, https://cloud.google.com/recaptcha-enterprise/docs/integrate-account-verification#understanding_the_configuration_process_of_mfa
+  public static void createMfaAssessment(
```

Comment:
nit: for better testability, I propose to return a status `true` or `false` instead of throwing exceptions. The printing to `stdout` is nice visual but testing the code sample by validating the return value is much simpler than hijacking `stdout` and matching the strings.